### PR TITLE
Add non-admin bigtable configs

### PR DESCRIPTION
**Pull request by geigerj**
Includes service, artman, GAPIC configs with hand-edits to required fields in the GAPIC config.

Note that this does not address the Bigtable admin APIs.

### CODE REVIEW

File: google/bigtable/bigtable.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Google Bigtable API service configuration
+
+type: google.api.Service
+config_version: 0
```
Comment (garrettjonesgoogle) [inline google/bigtable/bigtable.yaml]: I just noticed that we're using different config versions all over; pubsub is using version 1, logging is using version 2, and you have version 0 here. Should we settle on one version? (We have only one version of code in toolkit reading these files)

File: google/bigtable/v2/bigtable_gapic.yaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
```
Comment (garrettjonesgoogle) [inline google/bigtable/v2/bigtable_gapic.yaml]: sidenote, unrelated to this review - we should get rid of this setting, it doesn't actually impact anything any more.

File: google/bigtable/bigtable.yaml (hunk 1 above)
Comment (geigerj) [inline google/bigtable/bigtable.yaml]: These are derived from the Google-internal source of truth; I think they have effects in API compiler but am not positive.

Comment (garrettjonesgoogle) [inline google/bigtable/bigtable.yaml]: Ok. We can leave them as-is.

**Comment by garrettjonesgoogle** [comment]
LGTM

File: google/bigtable/bigtable.yaml (hunk 1 above)
Comment (bjwatson) [inline google/bigtable/bigtable.yaml]: They're also a way of tracking if we have the latest version from Google.

File: google/bigtable/v2/bigtable_gapic.yaml (hunk 2 above)
Comment (bjwatson) [inline google/bigtable/v2/bigtable_gapic.yaml]: ACK

Comment (bjwatson) [inline google/bigtable/v2/bigtable_gapic.yaml]: See https://github.com/googleapis/googleapis/pull/111

**Pull request merged**